xbmc/interfaces/legacy/Player.cpp

```diff
@@ -426,26 +426,26 @@ namespace XBMCAddon
       }
     }
 
-    std::vector<String>* Player::getAvailableSubtitleStreams()
+    std::vector<String> Player::getAvailableSubtitleStreams()
```

Comment:
Yeah. I'm just wondering why the return type was changed from a pointer (and if that even works). I guess I was assuming someone thought it was a leak - which I don't think it is but I was trying to figure out why someone might have